### PR TITLE
Add origin offline/ORIGIN_LOGGED_OUT to troubleshooting

### PR DESCRIPTION
**Pull request by itscynxx**
i included these together because they tend to go hand in hand with solutions. sometimes origin offline is due to not being logged in and sometimes logged out is due to needing to restart the task manager process (ea service try to be coherent challenge)

request for this came from a [thread on the discord server](https://discord.com/channels/920776187884732556/1140985677937377300)

### CODE REVIEW

File: docs/installing-northstar/troubleshooting.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ If the usual workarounds of restarting EA/Origin, rebooting your pc, or running
 * Sign out and exit EA/Origin, then start Northstar. It will prompt you for a login, _hopefully_ fixing it if nothing else worked
 * Start normal Titanfall 2 and then Northstar (_ONLY WORKS SOMETIMES_)
 
+## Origin Offline/ORIGIN_LOGGED_OUT <a href="#origin-offline" id="origin-offline"></a>
+
+This error usually happens when EA doesn't properly connect/see the EA App process on your pc. Most of the time this can be fixed by:
+
+1. Making sure you're signed into the EA App and can see Titanfall in your EA Library (only applies to Steam owners. If you can't, check [EA's official guide for linking accounts](https://help.ea.com/en/help/pc/link-ea-and-steam/)
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
1. Making sure you're signed into the EA App and can see Titanfall2 in your EA Library (only applies to Steam owners. If you can't, check [EA's official guide for linking accounts](https://help.ea.com/en/help/pc/link-ea-and-steam/)
```

File: docs/installing-northstar/troubleshooting.md [hunk 2]
```diff
@@ -17,6 +17,15 @@ If the usual workarounds of restarting EA/Origin, rebooting your pc, or running
...
+2. Using task manager to end the `EA App` and `EA Background service` tasks, opening the EA App, then launching Northstar
+
+If you don't know how to use task manager, press `"ctrl + alt + delete"`, select Task Manager, and hit `"More Info"` on the bottom right. If you can't see more info, then you've already clicked it before and don't need to do it again. After this, when you right click on a process it will open a small pop up with `"End Task"` as an option, which is what you want to use.
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: `CTRL+SHIFT+ESC` exists btw ;)

Also I'm not sure if we should describe how to open task manager tbh... On one side it feels to basic, on the other side, it's probably best to just link an external (official) resource that explains how to open task manager to not clutter up the section.

File: docs/installing-northstar/troubleshooting.md [hunk 3]
```diff
@@ -17,6 +17,15 @@ If the usual workarounds of restarting EA/Origin, rebooting your pc, or running
 * Sign out and exit EA/Origin, then start Northstar. It will prompt you for a login, _hopefully_ fixing it if nothing else worked
 * Start normal Titanfall 2 and then Northstar (_ONLY WORKS SOMETIMES_)
 
+## Origin Offline/ORIGIN_LOGGED_OUT <a href="#origin-offline" id="origin-offline"></a>
+
+This error usually happens when EA doesn't properly connect/see the EA App process on your pc. Most of the time this can be fixed by:
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: Do you "connect to"?

**Review by GeckoEidechse — COMMENTED** [review]
Thanks for the PR <3

I left a few comments below

File: docs/installing-northstar/troubleshooting.md (hunk 2 above)
Comment (itscynxx) [reply docs/installing-northstar/troubleshooting.md]: i just copy pasted it from where we explain it later in the page, i can remove it from this pr and change both to link to an external resource

File: docs/installing-northstar/troubleshooting.md [hunk 4]
```diff
@@ -17,6 +17,13 @@ If the usual workarounds of restarting EA/Origin, rebooting your pc, or running
 * Sign out and exit EA/Origin, then start Northstar. It will prompt you for a login, _hopefully_ fixing it if nothing else worked
 * Start normal Titanfall 2 and then Northstar (_ONLY WORKS SOMETIMES_)
 
+## Origin Offline/ORIGIN_LOGGED_OUT <a href="#origin-offline" id="origin-offline"></a>
+
+This error usually happens when the EA App fails to connect to EA servers properly. To fix this, you can:
+
+1. Make sure you're signed into the EA App and can see Titanfall2 in your EA Library (only applies to Steam owners. If you can't, check [EA's official guide for linking accounts](https://help.ea.com/en/help/pc/link-ea-and-steam/)
+2. Use task manager to end the `EA App` and `EA Background service` tasks, open the EA App, then launch Northstar
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: Missing dot at the end of sentence

```suggestion
2. Use task manager to end the `EA App` and `EA Background service` tasks, open the EA App, then launch Northstar.
```

File: docs/installing-northstar/troubleshooting.md [hunk 5]
```diff
@@ -17,6 +17,13 @@ If the usual workarounds of restarting EA/Origin, rebooting your pc, or running
 * Sign out and exit EA/Origin, then start Northstar. It will prompt you for a login, _hopefully_ fixing it if nothing else worked
 * Start normal Titanfall 2 and then Northstar (_ONLY WORKS SOMETIMES_)
 
+## Origin Offline/ORIGIN_LOGGED_OUT <a href="#origin-offline" id="origin-offline"></a>
+
+This error usually happens when the EA App fails to connect to EA servers properly. To fix this, you can:
+
+1. Make sure you're signed into the EA App and can see Titanfall2 in your EA Library (only applies to Steam owners. If you can't, check [EA's official guide for linking accounts](https://help.ea.com/en/help/pc/link-ea-and-steam/)
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: Missing dot at the end of sentence

```suggestion
1. Make sure you're signed into the EA App and can see Titanfall2 in your EA Library (only applies to Steam owners. If you can't, check [EA's official guide for linking accounts](https://help.ea.com/en/help/pc/link-ea-and-steam/).
```

**Review by GeckoEidechse — COMMENTED** [review]
Missing punctuation. Gonna fix it myself rq.

**Pull request merged**